### PR TITLE
Make CRLF go away

### PR DESCRIPTION
**Pull request by UGisBusy**
i want to be included in the contribitor list so i did something. 🎉

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```
Comment (NOOBDY) [inline .gitattributes]: [It is good practice to leave eof newline](https://stackoverflow.com/a/729795)

**Review by NOOBDY — CHANGES_REQUESTED** [review]

**Review by Error0229 — DISMISSED** [review]
LGTM 👀👍

File: .gitattributes (hunk 1 above)
Comment (UGisBusy) [reply .gitattributes]: yes sir my lord

**Review by NOOBDY — APPROVED** [review]
LGTM

**Review by Error0229 — APPROVED** [review]
👁️👁️
        👍
LGTM

**Pull request merged**